src/Message/AbstractRequest.php

```diff
@@ -966,7 +995,10 @@ protected function isUpdate()
      * Set $validateCard to false to skip card validation.
      *
      * @param string $paymentMethodType default null
+     *
      * @return stdClass
+     * @throws InvalidRequestException
+     * @throws InvalidCreditCardException
```

Comment:
What part of this method throws an `InvalidCreditCardException`? Might be helpful to add a comment.

Reply:
in `buildPaymentMethod`